zkllvm/use-cases/zk-bridge/hashes.mdx

```diff
@@ -0,0 +1,299 @@
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+# Write a circuit with hashes
+
+This 'how-to' series constructs a 'mock' zero knowledge bridge from scratch. The series starts with writing a simple circuit that uses hashes and ends with creating an algorithm for state-proof verification.
```

Comment:
Maybe I'm wrong, but usually `zero-knowledge` contains hyphen when using as an adjective.

Reply:
You are 100% right, I've added the hyphen